opensearch/services/datatypes/datatypes.php

```diff
@@ -0,0 +1,430 @@
+					array_push($datatypeDetails_markup, "<a target='_blank' href=\"" . $datatypeDetail['uri'] . "\">" . $datatypeDetail['label'] . "</a>", "<br>" . $datatypeDetail['creationTime'], "<br>" . $datatypeDetail['lastModifiedTime'], "<br>" . $datatypeDetail['expectedUses']);
+				}
+			$html .= implode('; ', $datatypeDetails_markup);
+			$html .= "</span>";
+		}
+
+		// Source Standard
+		if (isset($result['sourcestandard'])) {
+			$html .= "<br /><span>Source Standard: ";
+			$sourceStandard_arr = explode(",", $result['sourcestandard']);
+			$sourceStandard_label_arr = explode(",", $result['sourceStandard_label']);
+			$sourceStandard_markup = array();
+			foreach ($group_arr as $i => $sourceStandard) {
+				array_push($sourceStandard_markup, "<a target='_blank' href=\"" . $sourceStandard . "\">" . $sourceStandard_label_arr[$i] . "</a>");
+			}
+			$html .= implode('; ', $sourceStandard_markup);
+			$html .= "</span>";
+		}
+
+		// Parameters
+		$parameters = $this->getParametersByDatatype($result['datatype']);
+		if (count($parameters) > 0) {
+			$html .= "<br /><span>Parameters: ";
+			if (count($parameters) > 0) {
+				$parameters_markup = array();
+				foreach ($parameters as $i => $parameter) {	
+					array_push($parameters_markup, "<a target='_blank' href=\"" . $parameter['uri'] . "\">" . $parameter['label'] . "</a>", "<br>" . $parameter['hasUnit']);
+				}
+				$html .= implode('; ', $parameters_markup);
+			}
+			$html .= "</span>";
+		}
+
+		// Subject Area
+		$subjectAreas = $this->getSubjectAreaByDatatype($result['datatype']);
+		if(count($subjectAreas) > 0){
+			$html .= "<br /><span>Subject Area: ";
+			$subjectAreas_markup = array();
+				foreach($subjectAreas as $key => $subjectArea){
+					array_push($subjectAreas_markup, $subjectArea['concept']);
+				}
+			$html .= implode('; ', $subjectAreas_markup);
+			$html .= "</span>";
+		}
+
+		// Datasets
+		$datasets = $this->getDatasetsByDatatype($result['datatype']);
+		if(count($datasets) > 0){
+			$html .= "<br /><span>Datasets: ";
+			$datasets_markup = array();
+				foreach($datasets as $key => $dataset){
+					array_push($datasets_markup, "<a target='_blank' href=\"" . $dataset['uri'] . "\">" . $dataset['label'] . "</a>");
+				}
+			$html .= implode('; ', $datasets_markup);
+			$html .= "</span>";
+		}
+
+		// access
+		if (isset($result['access'])) {
+			$html .= "<br /><span>Access restriction: " . $result['access'] . "</span>";
+		}
+
+		$html .= "</div>";
+		return $html;
+	}
+	
+	/**
+	* Return SPARQL query header component
+	* @param string $type search type (e.g. 'parameters', 'authors', 'keywords')
+	* @return string query header component (e.g. 'SELECT ?id ?label')
+	*/
+	public function getQueryHeader($type) {
+	
+		$header = "";
+		switch($type) {
+			case "datatypes":
+				$header .= "?datatype ?dco_id ?label ?year ?parameterLabel ?access ";
+				$header .= '(GROUP_CONCAT(DISTINCT ?creator ; SEPARATOR=",") AS ?creator) ';
```

Comment:
I am not sure we should be defining a new variable (i.e. ?creator, ?creator_label, ?sourceStandard, ?sourceStandard_label)  with the same name as the existing variable.

See [project.php](opensearch/services/projects/projects.php) as an example.